README.md

```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio
 
-> Description the project.
+> In this project I have developed a responsive portfolio based on figma design. Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
 
 
 ## Built With
```

Comment:
- [ ] Your ```Live Demo Link``` is not working, you can update it with a working link or remove it from the README, I think it will help make your readme professional.